tools/bumper/bumper.go

```diff
@@ -79,8 +79,11 @@ func main() {
 
 		if componentBumpNeeded {
 			logger.Printf("Bumping %s from %s to %s", componentName, currentReleaseTag, updatedReleaseTag)
-			// reset --hard git repo
-			exitWithError(fmt.Errorf("reset --hader repo not implemented yet"))
+
+			err = cnaoRepo.resetRepoInAllowedList(getAllowedList())
+			if err != nil {
+				exitWithError(errors.Wrapf(err, "Failed to reset bumping repo before bumping component %s", componentName))
+			}
```

Comment:
What do you think about moving the component bumping to a function so we can put the resetRepoInAllowedList into a `defer` so we don't need the last call to it ?

Reply:
is `handleBump` sound good?

Reply:
`handleBump` sound good?